schema.graphql

```diff
@@ -250,6 +250,36 @@ type PluginMetadata {
   createdAt: DateTime!
 }
 
+enum Zone {
+  INNER
+  OUTER
+}
+
+enum Section {
+  LEFT
+  CENTER
+  RIGHT
+}
+
+enum Area {
+  TOP
+  MIDDLE
+  BOTTOM
+}
+
+type Location {
```

Comment:
Rename it to WidgetLocation, otherwise Location is too abstract and confusing because it sounds like LatLng